config/etfuturum/functions.cfg

```diff
@@ -0,0 +1,278 @@
+# Configuration file
+
+##########################################################################################################
+# changes
+#--------------------------------------------------------------------------------------------------------#
+# Changes to existing content.
+##########################################################################################################
+
+changes {
+    # Adds a packet to send the attackedAtYaw field value to the client, which allows the screen to tilt based on where damage came from, and either left or right for direction-less sources like drowning or burning, instead of tilting to the left no matter what. [default: true]
+    B:enableAttackedAtYawFix=true
+
+    # Dead Bushes drop sticks [default: true]
+    B:enableBushSticks=true
+
+    # Backports recipe buffs to doors (from 1 to 3) [default: true]
+    B:enableDoorRecipeBuffs=true
```

Comment:
set to false, we dont use vanilla door recipes

Reply:
I have already introduced Gregified door recipes and disabled these recipes for GTNH in the EFR code. I'll set this to false to be doubly sure.